<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>430</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>490</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Default</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>160</width>
     <height>486</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="labelLayout">
    <property name="leftMargin">
     <number>30</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Projection Type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Line type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Edge color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Edge thickness</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>View method</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Vertex size</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Vertex color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Background</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>161</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>-1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QComboBox" name="projectionType">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Central</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Parallel</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="lineType">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Solid</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dotted</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="edgeColorBtn">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Set Color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="edgeThickness">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="showGroupSeparator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="vertexType">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>60</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Square</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Circle</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="vertexSize">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="vertexColorBtn">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Set Color</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backgroundColorBtn">
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Set Color</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>560</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>560</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
